packages/multichain/src/middlewares/MultichainSubscriptionManager.test.ts

```diff
@@ -151,9 +153,14 @@ describe('MultichainSubscriptionManager', () => {
     multichainSubscriptionManager.subscribe({ scope, origin, tabId });
     multichainSubscriptionManager.unsubscribeByOriginAndTabId(origin, tabId);
 
-    mockSubscriptionManager.events.on.mock.calls[0][1](
-      newHeadsNotificationMock,
-    );
+    expect(mockSubscriptionManager.destroy).toHaveBeenCalled();
+  });
+
+  it('should unsubscribe when the middleware is destroyed', () => {
```

Comment:
Should there be an assertion in this test about the state of subscriptions? Or I suppose you can't access this 🤔

Reply:
Like you said, it's private so you can't actually check the internal state